.github/workflows/build-dumb-pypi.yml

```diff
@@ -159,7 +159,8 @@ jobs:
       env:
         _ANSIBLE_SDIST_FROM_MAKEFILE: 1
```

Comment:
This is also no longer needed.
```suggestion
```

Reply:
And the above `packaging` and `straight.plugin` aren't needed either — it's all now encapsulated within the PEP 517 interface.

Reply:
4493469bf7040aa25896aea3780046ab8b0b1295.